sig-ml-developer-experience/README.md

```diff
@@ -0,0 +1,30 @@
+# ODH ML Developer Experience Special Interest Group
+
+SIG ML Developer Experience covers end-to-end experience for a data scientist, a data analyst or an application developer
+
+The [charter](./charter.md) defines the scope and governance of the ML Developer Experience Special Interest Group
+
+## Meetings
+[Monthly SIG meeting:](http://meet.google.com/ipx-ewwx-cup)  Every 4th Tuesday of the month
+[Meeting notes and agenda](https://docs.google.com/document/d/1cGK5kkhp6SLhCc__pmLCfHEiCP-lZPcxpJTsz1nwaNY/edit)
+
+## Leadership
+
+### Chairs
```

Comment:
Possibly dumb question - do we want to include the GitHub handles for all the members here?

Reply:
Both under chairs and members